samples/matter/light_switch/Kconfig

```diff
@@ -23,6 +23,23 @@ config STATE_LEDS
 	  the device into a network or the factory reset initiation. Note that setting this option to
 	  'n' does not disable the LED indicating the state of the simulated bolt.
 
+if MPSL_FEM
+
+config FEM_802_15_4_DEFAULT_TX_POWER
+	int "Set the default 802.15.4 output power (dBm) for device with FEM"
+	default 20
+	help
+	  Use this setting to set the default Thread (802.15.4) output power for device that
+	  uses FEM. This value has a unit in dBm and represents the Tx power at Antenna port of FEM device.
+
+if FEM_802_15_4_DEFAULT_TX_POWER != 0
+
+# see subsys/mpsl/fem for full definition
+config MPSL_FEM_NRF21540_TX_GAIN_DB
+	default 20
```

Comment:
```suggestion
	default FEM_802_15_4_DEFAULT_TX_POWER
```
?

Reply:
We probably can't use this setting because AFAIK OPEN_THREAD TX power is power at the antenna port, and MPSL_FEM_NRF21540_TX_GAIN_DB is a gain of the FEM module. So I don't think merging these values here is a good idea These default values are set to reach the output power value as previously.  But maybe you have some counterarguments?

Reply:
Ok, I see the problem. I didn't do it clearly :) I wanted to leave FEM setting to the default value defined in Zephyr and override it only when OPENTHREAD_TX_POWER is different than 0 because it will be helpful during testing. I thought that we could have a possibility to disable FEM by setting OPEN_THREAD_TX_POWER to 0 (The power form nRF21DK should be similar to nRF52DK), and in TAS, we can use this Kconfig to verify FEM instead of using an additional board as a reference. We should set power on SoC as low as possible due to the limitation of FEM input power. So, for example, OPENTHREAD_PWER is set to 10, FEM_GAIN is set to 20 dB, and as a result, SoC power is set to -10 dBm, and this config is the best for FEM and gives the lowest interferences as I've learned recently. We cannot set FEM to 10 and then set OPENTHREAD_POWER to 20 because the resulting SoC power will be set to +10 dBm, exceeding the maximum possible value for FEM input power (Which is equal to +5dBm). Due to that, I think we cannot merge these settings, and it is better to set FEM gain to 20 dB and control only OPENTHREAD_POWER, leaving 0 value for disabling FEM.